digihel/templates/base.html

```diff
@@ -105,10 +105,15 @@
         </footer>
 
         {# Global javascript #}
-        <script src="{% static 'jquery/dist/jquery.min.js' %}"></script>
-        <script src="{% static 'bootstrap-sass/assets/javascripts/bootstrap.min.js' %}"></script>
-        <script src="{% static 'matchHeight/jquery.matchHeight-min.js' %}"></script>
-        <script type="text/javascript" src="{% static 'js/digihel.js' %}"></script>
+        {% compress js %}
```

Comment:
This will cause a long delay in loading pages. It's better to compress just `displayEvents.coffee`.